### DISCUSSION TITLE
Theia IDE 1.54 preview testing

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#14230] [Release Announcements] [answered]
The new version 1.54.0 of the Theia IDE is available on the preview channel now, please join the preview testing! You can download it here: [Linux](https://download.eclipse.org/theia/ide-preview/1.54.0/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.54.0/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.54.0/windows/TheiaIDESetup.exe). You can update your existing installation by setting the preference `updates.channel` to `preview`. Please respond here when you were able to test the preview without finding blockers, by commenting with a :heavy_check_mark: . If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.

**Comment by tsmaeder** [comment]
Hi folks, can we get a go/no-go from all the testers by the end of the day?

**Comment by rschnekenbu** [comment]
✔️ I could not find any blocker for this 1.54 version, testing also the newly integrated AI feature

**Reply by tsmaeder** [reply]
Same here, now new bugs observed in using the IDE today.

**Reply by JonasHelming** [reply]
"Now" or "No" :-)

**Reply by eneufeld** [reply]
Wasn't able to test much yet, but I didn't encounter any issues neither

**Comment by sgraband** [comment]
:heavy_check_mark: I do not see any new issues for this version. AI Feature works great!

**Reply by jfaltermeier** [reply]
✔️

**Comment by JonasHelming** [comment]
✔️

**Comment by tsmaeder** [comment]
I think we have a quorum for promoting the build to production. @sgraband will you do it?

**Reply by sgraband** [reply]
Yes i will go ahead and trigger the promotion :+1:

**Comment by sgraband** [comment, marked as answer]
1.54.0 is promoted to the stable channel
